.github/workflows/release.yml

```diff
@@ -216,6 +174,49 @@ jobs:
           if [ "${{ inputs.dry_run }}" = "true" ]; then  
             box_summary="[Dry run] "
           fi
+      - name: Promote artifacts
```

Comment:
Give an extra line to separate these 2 steps

Reply:
added extra line